example/cajita_tutorial/CMakeLists.txt

```diff
@@ -13,3 +13,4 @@ add_subdirectory(01_types)
 add_subdirectory(02_global_mesh)
 add_subdirectory(03_partitioner)
 add_subdirectory(04_global_grid)
+add_subdirectory(05_fast_fourier_transform)
```

Comment:
You'll need to wrap this in a check for whether heffte enabled

Reply:
thanks 👍 
Should we also have a `if(Cabana_ENABLE_MPI)` around the couple MPI tests?

Reply:
No, Cajita always requires MPI